GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.cs

```diff
@@ -26,321 +26,217 @@ public partial class Dashboard : GitModuleControl
         private readonly TranslationString issues = new TranslationString("Issues");
         private readonly TranslationString openRepository = new TranslationString("Open repository");
         private readonly TranslationString translate = new TranslationString("Translate");
-        private readonly TranslationString directoryIsNotAValidRepositoryCaption = new TranslationString("Open");
-        private readonly TranslationString directoryIsNotAValidRepository = new TranslationString("The selected item is not a valid git repository.\n\nDo you want to abort and remove it from the recent repositories list?");
-        private readonly TranslationString directoryIsNotAValidRepositoryOpenIt = new TranslationString("The selected item is not a valid git repository.\n\nDo you want to open it?");
-        private readonly TranslationString _showCurrentBranch = new TranslationString("Show current branch");
```

Comment:
These are no longer relevant. Do I need to remove all referenced translations myself or will it be done automatically by the translation app?